src/application.js

```diff
@@ -13,21 +13,22 @@ import BundleDownloader from './workers/bundle_downloader';
 import BundleFinaliser from './workers/bundle_finaliser';
 
 class Application extends Builder {
-  constructor(logger) {
+  constructor(serverLogger, backgroundLogger) {
```

Comment:
What's the difference with the serverLogger and the backgroundLogger? Can they share the same log object?

Reply:
Server logger writes to the file and outputs jsons